webhook/resourcesemantics/conversion/conversion.go

```diff
@@ -109,17 +110,26 @@ func (r *reconciler) convert(
 	out := outZygote.DeepCopyObject().(ConvertibleObject)
 
 	hubGVK := inGVK.GroupKind().WithVersion(conv.HubVersion)
-	ctx = logging.WithLogger(ctx, logger.With(
+
+	logger = logger.With(
 		zap.String("inputType", formatGVK(inGVK)),
 		zap.String("outputType", formatGVK(outGVK)),
 		zap.String("hubType", formatGVK(hubGVK)),
-	))
+	)
 
 	// TODO(dprotaso) - potentially error on unknown fields
 	if err = json.Unmarshal(inRaw.Raw, &in); err != nil {
 		return ret, fmt.Errorf("unable to unmarshal input: %w", err)
 	}
 
+	if acc, err := kmeta.DeletionHandlingAccessor(in); err == nil {
```

Comment:
I wonder if the convertible interface should implement 
`metav1.Object` which would simplify all of this

but that's another issue

Reply:
yeah, that'd help stuff.
But I guess different PR, yes :)

Reply:
sure

Reply:
Also I can ignore the error, since I guess anything that can be in the webhook version conversions has to be a k8s object :)